packages/foundry/src/create.ts

```diff
@@ -0,0 +1,80 @@
+import { basename } from "node:path";
+
+import { cloneProjectTemplate } from "./cloneProjectTemplate.js";
+import { replaceTokens } from "./replaceTokens.js";
+
+const BaseRepositoryAddress = "workleap/wl-foundry-cli/templates";
+
+type TemplateId = "host-application" | "remote-module" | "static-module";
+
+interface TemplateDetails {
+    repositoryUrl: string;
+    action: (outputDirectory: string, options: Record<string, string>) => Promise<void>;
+}
+
+const getName = (outputDirectory: string) => {
+    return basename(outputDirectory);
+};
+
+const getScope = (options: Record<string, string>, flagName: string) => {
+    let scope = options[flagName] as string;
+
+    if (scope) {
+        if (scope[0] !== "@") {
```

Comment:
This behavior will need to be validated by path. I'm not sure we really require the scope to start with a `@`